e2e/atlas/events_test.go

```diff
@@ -38,6 +38,7 @@ func TestEvents(t *testing.T) {
 		cmd := exec.Command(cliPath,
 			atlasEntity,
 			eventsEntity,
+			"project",
```

Comment:
you can use `projectEntity` defined in https://github.com/mongodb/mongocli/blob/6ac1f831b2a7d4ae60f4a7d025e4ab3c8da50d79/e2e/atlas/helper_test.go#L46
```suggestion
			projectEntity,
```